tests/integration/conftest.py

```diff
@@ -1055,9 +1084,37 @@ def config_transform(self) -> Callable[[WorkspaceConfig], WorkspaceConfig]:
         return lambda wc: wc
 
     @cached_property
-    def include_object_permissions(self) -> None:
+    def include_object_permissions(self) -> list[str] | None:
         return None
 
+    def configure_include_object_permissions(self) -> None:
+        # Initialize include_object_permissions for the created fixtures
+        # Currently only supports the object types for which the fixtures exist
+
+        if not self.config.include_object_permissions:
+            self.config.include_object_permissions = []
```

Comment:
Please create a new attribute for the `include_object_permissions` and added to the config like `include_databases` and `created_databases`